pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -64,6 +64,11 @@ in buildPythonPackage {
     runHook postConfigure
   '';
 
+  # This patch from Arch Linux fixes Cura segfaulting on startup
+  # https://github.com/Ultimaker/Cura/issues/3438
+  # It can probably removed on 5.10.3
+  patches = [ ./pyqt5-cura-crash.patch ];
```

Comment:
We try to avoid local copies of available patches in nixpkgs. Please  `fetchpatch` this patch from the Arch Linux git repo instead:
https://git.archlinux.org/svntogit/packages.git/plain/repos/extra-x86_64/pyqt5-cura-crash.patch?id=6cfe64a3d1827e0ed9cc62f1683a53b582315f4f ,
see other examples in nixpkgs.

Reply:
Fixed.